plugins/catalog-backend-module-gitlab/src/lib/client.ts

```diff
@@ -107,12 +115,92 @@ export class GitLabClient {
           },
           body: JSON.stringify({
             variables: { group: groupPath, endCursor },
-            query: `query($group: ID!, $endCursor: String) {
+            query: `query listDescendantGroups($group: ID!, $endCursor: String) {
+              group(fullPath: $group) {
+                descendantGroups(first: 100, after: $endCursor) {
+                  nodes {
+                    id
+                    name
+                    description
+                    fullPath
+                    parent {
+                      id
+                    }
+                  }
+                  pageInfo {
+                    endCursor
+                    hasNextPage
+                  }
+                }       
+              }
+            }`,
+          }),
+        },
+      ).then(r => r.json());
+      if (response.errors) {
+        throw new Error(`GraphQL errors: ${JSON.stringify(response.errors)}`);
+      }
+
+      if (!response.data.group?.descendantGroups?.nodes) {
+        this.logger.warn(
+          `Couldn't get groups under ${groupPath}. The provided token might not have sufficient permissions`,
+        );
+        continue;
+      }
+
+      const groupsData = response.data.group.descendantGroups.nodes;
+
+      for (let i = 0; i < groupsData.length; i++) {
+        const groupItem = groupsData[i];
+
+        const formattedGroupResponse = {
+          id: Number(groupItem.id.replace(/^gid:\/\/gitlab\/Group\//, '')),
+          name: groupItem.name,
+          description: groupItem.description,
+          full_path: groupItem.fullPath,
+          parent_id: Number(
+            groupItem.parent.id.replace(/^gid:\/\/gitlab\/Group\//, ''),
+          ),
+        };
+
+        items.push(formattedGroupResponse);
+      }
+      ({ hasNextPage, endCursor } =
+        response.data.group.descendantGroups.pageInfo);
+    } while (hasNextPage);
+    return { items };
+  }
+
+  async getGroupMembers(
+    groupPath: string,
+    relations: string[],
+  ): Promise<PagedResponse<GitLabUser>> {
+    const items: GitLabUser[] = [];
+    let hasNextPage: boolean = false;
+    let endCursor: string | null = null;
+    do {
+      const response: GitLabGroupMembersResponse = await fetch(
+        `${this.config.baseUrl}/api/graphql`,
+        {
+          method: 'POST',
+          headers: {
+            ...getGitLabRequestOptions(this.config).headers,
+            ['Content-Type']: 'application/json',
+          },
+          body: JSON.stringify({
+            variables: { group: groupPath, relations: relations, endCursor },
+            query: `query getGroupMembers($group: ID!, $relations: [GroupMemberRelation!], $endCursor: String) {
```

Comment:
do this
```suggestion
            query: /* GraphQL */ `query getGroupMembers($group: ID!, $relations: [GroupMemberRelation!], $endCursor: String) {
```
and then run `yarn prettier -w plugins/catalog-backend-module-gitlab/src/lib/client.ts`